python/poetry.lock

```diff
@@ -1,4 +1,4 @@
-# This file is automatically @generated by Poetry 1.7.1 and should not be changed by hand.
+# This file is automatically @generated by Poetry 1.5.1 and should not be changed by hand.
```

Comment:
This seems generated by an old version of poetry. Seems OK anyways?

Reply:
Fixed, I had 2 poetry installs on my VM